Data_Engine/Create/NTree.cs

```diff
@@ -109,14 +109,14 @@ public static NTree<T> NTree<T>(IEnumerable<Leaf<T>> leaves, int splits = 16)
         private static Output<List<Leaf<T>>, List<Leaf<T>>> SplitList<T>(List<Leaf<T>> list, int sampleSize = 60)
         {
             // find centre
-            NBound box = new NBound();
+            NBound box = null;
 
             //sampleSize = Math.Max(2, sampleSize);
             sampleSize = Math.Min(list.Count, sampleSize);
 
             int step = (int)Math.Floor((decimal)(list.Count / sampleSize));
             for (int i = 0; i < sampleSize; i++)
-                box += (NBound)list[i * step].Bounds;
+                box += list[i * step].Bounds;
```

Comment:
Does this work in testing @kThorsager ? The box being `null` on the first path would be doing a calculation of `null + box` which I wouldn't expect to work initially?

Reply:
You'd think so but I was wrote the `+` operator to simply return the second box if the first one is `null`, so yes.

That should perhaps be changed however

Reply:
It was confusing code so I changed it